chain/chain/src/sharding.rs

```diff
@@ -13,6 +15,18 @@ pub fn shuffle_receipt_proofs<ReceiptProofType>(
     receipt_proofs.shuffle(&mut rng);
 }
 
+pub fn cares_about_shard_this_or_next_epoch(
```

Comment:
nit: As I read this function, it might as well be a member of shard tracker.
Feel free to ignore